string input_text
---
bool success
